### DISCUSSION TITLE
Candidate_document as a link

### DISCUSSION THREAD

**Discussion by tlustopro** [manykarim/robotframework-doctestlibrary#85] [Q&A]
Hi!
Can i use `PDF Should Contain Strings` keyword only with downloaded PDF file, or this kw also works with file saved on the server? I would like to provide URL with PDF file instead of path to document saved on my disk.
Thanks!

**Comment by manykarim** [comment]
Hey there,
the current version (0.18.1) expects a local file path for `PDF Should Contain Strings`.
But it would only be a minor change to allow also URLs (like I do in the `Compare Images`). I guess it would even be more consistent that way. Let me update the library in the next days.
Would you mind trying it out in a preview release, once I implemented it?

**Reply by tlustopro** [reply]
Hi Many,
thanks for fast reply. 
Sure, ping me once this will be implemented and i will try it for you!

**Comment by manykarim** [comment]
Created https://github.com/manykarim/robotframework-doctestlibrary/issues/86

**Reply by manykarim** [reply]
Took a bit longer, but URL support for PDF and PrintJob Keywords has been implemented in v0.19.0
https://github.com/manykarim/robotframework-doctestlibrary/issues/86
